dynamics-nav-app/LocalFunctionality/Germany/how-to-print-test-reports-for-delivery-reminders.md

```diff
@@ -15,7 +15,7 @@
     ms.author: edupont
 
 ---
-# How to: Print Test Reports for Delivery Reminders
+# How to: Print Test Reports for Delivery Reminders Germany
```

Comment:
Add parentheses to be consistent with title.

Reply:
Done.